builder.json

```diff
@@ -6,7 +6,7 @@
     "gradlew": "{source_dir}/gradlew"
   },
   "pre_build_steps": [
-    "cd {source_dir}",
```

Comment:
could you add log (plain echo) here, why are we doing this for posterity.

Reply:
and put a link to https://github.com/awslabs/aws-sdk-swift/issues/808

Reply:
Done